legal-api/src/legal_api/services/filings/validations/validation.py

```diff
@@ -190,6 +191,9 @@ def validate(business: Business,  # pylint: disable=too-many-branches,too-many-s
                 elif k == Filing.FILINGS['putBackOff'].get('name'):
                     err = put_back_off_validate(business, filing_json)
 
+                elif k == Filing.FILINGS['transparencyRegister'].get('name'):
```

Comment:
I don't see a schema for this. Yet to implement?

Reply:
I mentioned this in my PR desc - just fixing the test cases before I PR it